libs/net/dal/Services/ContentService.cs

```diff
@@ -265,6 +266,25 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWi
                 .Query(filter.Keyword.ToLower())
             ));
 
+        if (!string.IsNullOrWhiteSpace(filter.Names))
+        {
+            var names = filter.Names.Split(',').Select(a => a.Trim()).ToList();
+            List<string> nameQueries = new();
+            foreach (var name in names)
+            {
+                var nameSegments = name.Split(' ');
+                var firstName = nameSegments.First();
+                var firstInitial = firstName.Substring(0, 1);
+                var lastName = nameSegments.Last();
```

Comment:
This logic can fail under certain conditions.
`"firstname"` would be a valid but would result in `firstName == lastName`.

Reply:
Are you referring to a scenario where the `name` of the individual is a single word like "Beyonce"?

Reply:
yes